index.ts

```diff
@@ -4,6 +4,12 @@ import axios from "axios";
 import * as dotenv from "dotenv";
 dotenv.config();
 
+class AlertError extends Error {
+  constructor(message: string) {
+    super(message);
+  }
+}
```

Comment:
You don't need to have all of this, you can only write: `class AlertError extends Error {}`